app/models/custom_organisation_name.rb

```diff
@@ -1,3 +1,10 @@
 class CustomOrganisationName < ApplicationRecord
-  validates :name, presence: true, uniqueness: { case_sensitive: false }
+  validate :custom_org_in_register?
+  validates :name, presence: true
```

Comment:
Intentional to remove the uniqueness validation?

Reply:
Yeah as the 'custom_org_in_register?' does this for us